docs/SRS/SRS.tex

```diff
@@ -365,24 +370,186 @@ \section{Project Issues}
+\subsubsection{Planning of the Development Phases}
+
+The development of the project will be conducted in two phases:
+
+\begin{enumerate}
+	\item Initial development of application and documentation
+	\item Refinement of application and documentation
+\end{enumerate}
+
+Phase 1 is where the bulk of the application will be designed and implemented. The design of both
+the components of the application and how the components will interact will de developed.
+Additional, Phase 1 is where most of the documentation and report will be written. Phase 1 will end
+with the Revision 0 Demonstration. Here the stakeholders will see the application implementation
+and be able to provide feedback.
+
+Phase 2 will be focused on refining the application and the documentation. There is expected to be
+no new major feature development and instead, all efforts will be focused on incorporating the
+stakeholders' feedback into the application.
 
 \subsection{Migration to the New Product}
+\subsubsection{Requirements for Migration to the New Product}
+
+There are no requirements for migrating to the new product.
+
+\subsubsection{Data That Has to Be Modified or Translated for the New System}
+
+No data needs to be modified or translated to the new system.
 
 \subsection{Risks}
 
+\begin{itemize}
+	\item Failures in the work orders and quotes work flow may lead to data loss.
+	\item Failures in the appointments work flow may lead to a loss in appointment or conflicting
+	      appointments.
+	\item Failure to meet deadlines will cause set backs in project's timeline. In the event of this, lower
+	      priority requirements may need to be dropped.
+\end{itemize}
+
 \subsection{Costs}
 
+There are no financial costs associated with the development of this application. All software and
+cloud infrastructure used are free to use. There will be about six months of development time
+required.
+
 \subsection{User Documentation and Training}
+\subsubsection{User Documentation Requirements}
+
+The application will feature a ``Getting Started'' guide, where it shall guide the user through the
+most common use cases. For vehicle owners, the use cases will include: searching for shops,
+requesting quotes, and scheduling appointments. For automotive shops, the use cases include:
+setting shop details, managing appointments, managing employees, responding to quotes, and managing
+work orders.
+
+\subsubsection{Training Requirements}
+
+Knowledge of how to navigate a website will be required. Documentation concerning detailed usage of
+the website's user flows will be provided to the user.
 
 \subsection{Waiting Room}
 
+There are currently no requirements that are not part of the initial release.
+
+% todo: could be a good place to put the stretch goals
+
 \subsection{Ideas for Solutions}
 
+% todo: ask the rest of the team on what they would add here
+
+During the requirements collection and understanding phase, some ideas with regard to what tooling
+will be used have occurred.
```

Comment:
> Ideas for solutions are obviously not requirements, but it is impossible when gathering requirements not to get ideas for how they might be implemented. Rather than discarding the ideas or—even worse—writing them as if they are requirements, a practical idea is to simply set aside a section of your specification for these ideas. Record each one faithfully in this area, and then hand them over, along with the requirements, to your designer. 

@TimChoy @LeonSo7 @joyxiao99
Is there any other ideas you guys had for the implementation?